features/ship/always_merge/current_branch/undo_warning.feature

```diff
@@ -0,0 +1,22 @@
+Feature: git-town undo prints a warning message for a merge commit
+
+  Background:
+    Given a Git repo with origin
+    And the branches
+      | NAME    | TYPE    | PARENT | LOCATIONS     |
+      | feature | feature | main   | local, origin |
+    And the current branch is "feature"
+    And the commits
+      | BRANCH  | LOCATION      | MESSAGE        |
+      | feature | local, origin | feature commit |
+    And Git setting "git-town.ship-strategy" is "always-merge"
+    # "I ran" verifies the exit status.
+    When I ran "git-town ship -m 'feature done'"
```

Comment:
I think the `I ran` step is part of the `Arrange` phase. In Gherkin (the language used in these Cucumber-based end-to-end tests) the `When` keyword is for steps in the `Act` phase.

```suggestion
    And I ran "git-town ship -m 'feature done'"
```

Reply:
Done.